### ISSUE TITLE
Reply to Comment emoji placement - move them above responses?

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#5226]
This is a small design issue we probably need to brainstorm a tiny bit before making a change, but the emoji shown when responding to a comment appear below the comments' responses:

![image](https://user-images.githubusercontent.com/24359/54768139-f5c52a00-4bd4-11e9-8c90-6efad5e9e539.png)

@ViditChitkara what do you think about this?

**Comment by ViditChitkara** [comment]
Maybe we can move the 'reply to comment' text box out of the main comment box??
Also if there are a lot of replied comments, the main comment box will be too big.
Any ideas?

**Comment by CleverFool77** [comment]
Hi 
Can we close the comments ..like it happens in threaded comments. If a person will click,then only it will open the replies.

**Comment by CleverFool77** [comment]
And the emoji should be above. :thinking:

**Comment by ViditChitkara** [comment]
That's a good idea, however if we want that the entire discussion should be available at a glance, will it be a good idea to manually view all replies? Just asking for a possibility.
Also, GitHub auto loads comments as we scroll down. For this maybe we'll have to somehow move the replied comments out and will have to find a way that they appear to be linked to the parent comment.

**Comment by ViditChitkara** [comment]
@CleverFool77 , @jywarren , what do you think?

**Comment by CleverFool77** [comment]
How about paginate for more comments ?

**Comment by jywarren** [comment]
:-) these are all cool ideas! Let's try to make a list of them, but let's
also think about ways to do this that would minimize the possibility of odd
errors and bugs -- the more complex it is, the harder it will be to
maintain, you know? Like, we could make everything collapsible with a <
button but make it shown by default, not hidden by default, so if something
JavaScript breaks, the page is not stuck with everything hidden.
Progressive loading also could be complex to maintain, and we don't yet
have super long comment threads, so perhaps we could think about this with
future React ideas for manage page state.

Also, interested in getting System Tests to work so that we can have robust
testing for JS features, because we are often seeing small JS issues like
the comment image uploading: https://github.com/publiclab/plots2/issues/5229
that are hard to detect automatically.

System tests will give us more reliability and confidence to do more
complex client-side javascript stuff without worrying that it'll keep
breaking! https://github.com/publiclab/plots2/pull/4888

For the moment I'd like to fix the comment image uploading and this emoji
placement, then look forward to new features. Thank you all so much!

**Comment by IshaGupta18** [comment]
Yes I think you are right @jywarren. Even I think lazy loading will be hard to maintain. The collapse idea is a good one and is simple and standard to execute. I think once these issues are fixed, we can move to more creative ideas. I would love to help out here!
